### PR TITLE
Abstracts, general and canonical type for HM

### PR DESCRIPTION
**Pull request by RathishKJ**

### CODE REVIEW

File: ontology/yaml/resources/HVAC/entity_types/GENERALTYPES.yaml [hunk 1]
```diff
@@ -357,4 +357,10 @@ GTWS:
   opt_uses:
   - thermal_power_capacity
   - differential_pressure_specification
-  - flowrate_requirement
\ No newline at end of file
+  - flowrate_requirement
+  
+HM:
+  description: "Tag for BTU meters"
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/GENERALTYPES.yaml:363]: heat, not BTU -- thats a unit of measurement.

**Review by tasodorff — CHANGES_REQUESTED** [review]
put this in the METERS namespace and reuse concepts there. i think this whole PR duplicates stuff that is already in there.

File: ontology/yaml/resources/HVAC/entity_types/GENERALTYPES.yaml (hunk 1 above)
Comment (RathishKJ) [reply ontology/yaml/resources/HVAC/entity_types/GENERALTYPES.yaml]: changed to "heat"

File: ontology/yaml/resources/HVAC/entity_types/HM.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+########################
+### Canonical Types  ###
+########################
+
+## BTU Meter ##
```
[the remark this answers is not in the document]
Comment (RathishKJ) [reply ontology/yaml/resources/HVAC/entity_types/HM.yaml]: Moved to Meters namespace

**Comment by RathishKJ** [comment]
Closing the pull request since we used the existing canonical HM_CHW.

**Pull request closed without merging**